ament_copyright/ament_copyright/__init__.py

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import pkg_resources
+import importlib_metadata
```

Comment:
@clalancette `importlib_metadata` is `importlib.metadata` on Python3.8 (see [here](https://importlib-metadata.readthedocs.io/en/latest/)), and so this script is failing for me on Windows.

Reply:
I added importlib-metadata in the `pip install` list for Rolling: https://index.ros.org/doc/ros2/Installation/Rolling/Windows-Development-Setup

It's somewhat redundant (since you can get it via `importlib.metadata` as you say), but it allows us to support Ubuntu 18.04 without a `try/except` block in the code.